### ISSUE TITLE
retrieve information about single subvolumes (btrfs subvolume list PATH_TO_SUBVOLUME)

### ISSUE THREAD

**Issue by serpedon** [btrfs8-revamp/btrfs8-revamp#12]
I would like to have a possibility get information only about single subvolumes. Currently, using 
```
btrfs subvolume list PATH_TO_SUBVOLUME
```
has the disadvantage that it lists ALL subvolumes, not only a specific one. Listing ALL subvolumes is annoying if one is interested only in a specific one, especially if there are many subvolumes, e.g. because of many snapshots. [Using ```grep``` or similar to select the correct line is error-prone.]

One solution would be to add a new command line switch to filter out only a single subvolume, e.g.
```
btrfs subvolume list -1 PATH_TO_SUBVOLUME
```
or 
```
btrfs subvolume list -1 PATH_TO_SUBVOLUME PATH
```
FYI: The last transaction id is used e.g. in [btrfs-snap](https://github.com/jf647/btrfs-snap) to check if a snapshot is needed at all. Currently we are using 
```
btrfs subvolume find-new PATH_TO_SUBVOLUME VERY_LARGE_NUMBER
```
but this is likely unstable and feels like a misuse of the command ```find-new```.

**Comment by moviuro** [comment]
Isn't this one similar to #11 ?

**Comment by serpedon** [comment]
Yes, it is similar. The main difference is that I am looking for the information of only a single subvolume whereas ```btrfs subvolume list``` seems to be designed to provide all available information.

I should note that ```btrfs subvolume show PATH_TO_SUBVOLUME``` already gives most/all information but the output is inconsistent (concerning format) with the output of ```btrfs subvolume list```

**Comment by moviuro** [comment]
Inconsistent output is also a discussed topic here #6 
I'm going to add the current man pages to the repo and we'll write in there what we expect the different commands to print.
This may take a while to adapt to .md style.
